master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Customizing the manifests
+title: Customizing the manifests (advanced)
```

Comment:
Don't know if we want to characterize this as "advanced" - it's probably a normal thing to do.